module_4/src/service/model.js

```diff
@@ -0,0 +1,43 @@
+export class ModelService {
```

Comment:
not sure if in this case you need a class/service. Here you have tree independent functions which are not actually tied to any model...

Reply:
so, it will be enough to use these functions without any class